src/percy-agent-client/dom.ts

```diff
@@ -110,20 +112,38 @@ class DOM {
    *
    */
   private serializeInputElements(clonedDOM: HTMLDocument): void {
-    const formNodes = this.originalDOM.querySelectorAll('input, textarea')
+    const formNodes = this.originalDOM.querySelectorAll(FORM_ELEMENTS_SELECTOR)
     const formElements = Array.from(formNodes) as HTMLFormElement[]
 
     formElements.forEach((elem) => {
       const inputId = elem.getAttribute('data-percy-element-id')
       const selector = `[data-percy-element-id="${inputId}"]`
-      const cloneEl = clonedDOM.querySelector(selector) as HTMLInputElement
+      const cloneEl = clonedDOM.querySelector(selector) as any
```

Comment:
I called it out in the commit that made this change, but want to be clear here too.

I made this any since we're looping over an array of DOM nodes that can be any of these (at this time): `select`, `input`, `textarea`

Since TS doesn't allow you to recast a type, I'd rather make this `any` since the type of node can/will change with each loop.